### PR TITLE
Red feature

### PR DESCRIPTION
**Pull request by mpinol**
Add red teams questions and tips @emiliza @vduvnjak

### CODE REVIEW

**Comment by nrichards** [comment]
@mbirk RED TEAM pull request is better than the rest

**Comment by emiliza** [comment]
plz fix merge conflicts, guyssss

**Comment by nrichards** [comment]
@emiliza RED TEAM pull request is better than the rest !!!

**Comment by erich-unity** [comment]
tru dat.

**Comment by emiliza** [comment]
green team refuses to review anything that can't be merged in cleanly.

**Comment by mbirk** [comment]
Wow, almost looks like more commits than lines of code changed ;)

File: questions.md [hunk 1]
```diff
@@ -4,4 +4,12 @@ Questions and answers
 - Q:Why use rebase instead of merge?  
 A: It depends on how you want to read your repositories history.  
 Reading the commits? -> rebase  
-Reading the pull requests -> merge
\ No newline at end of file
+Reading the pull requests -> merge
+
+
+- Q:How do I delete a branch? :( Is it git branch -d branch-name?
```
Comment (emiliza) [inline questions.md:9]: Do you want to delete from remote or local?

File: setup.md [hunk 2]
```diff
@@ -16,6 +16,13 @@ git config --global user.name "John Doe"
 git config --global user.email johndoe@example.com
 ```
 
+Important (optional) tip
+------------
+- Color is easier to read
+```
+git config color.ui true
```
Comment (mbirk) [inline setup.md:23]: Why not `git config --global` ?

**Comment by mbirk** [comment]
Middling quality, but it's okay :P

**Comment by emiliza** [comment]
DEAR RED TEAM PLEASE FIX CONFLICTS <3

**Comment by chaqke** [comment]
please use pull --rebase to avoid generating unnecessary merge commits ;-)

**Comment by emiliza** [comment]
:(

**Pull request closed without merging**